bokehjs/src/lib/models/annotations/box_annotation.ts

```diff
@@ -773,24 +773,23 @@ export class BoxAnnotationView extends AnnotationView implements Pannable, Pinch
     } = this.model
 
     switch (target) {
-      case "top_left":     return this._handles.top_left == null     ? tl_cursor : null
-      case "top_right":    return this._handles.top_right == null    ? tr_cursor : null
-      case "bottom_left":  return this._handles.bottom_left == null  ? bl_cursor : null
-      case "bottom_right": return this._handles.bottom_right == null ? br_cursor : null
-      case "left":         return this._handles.left == null         ? ew_cursor : null
-      case "right":        return this._handles.right == null        ? ew_cursor : null
-      case "top":          return this._handles.top == null          ? ns_cursor : null
-      case "bottom":       return this._handles.bottom == null       ? ns_cursor : null
+      case "top_left":     return this._handles.top_left == null     ? tl_cursor : this._handles.top_left.tl_cursor
```

Comment:
Was there a reason to return null for the cursor if the corresponding handle was defined? The result of returning null here is that when the user hovers the mouse over a handle they get one of the resize cursors, but when they start dragging, the cursor goes back to the default. I think that the cursor should not change while the user is dragging the handle.